app/logic/spreadsheet.py

```diff
@@ -158,8 +160,8 @@ def calculateRetentionRate(fall_dict, spring_dict):
             retention_rate = len(fall_participants & spring_participants) / len(fall_participants)
         except ZeroDivisionError:
             pass
-
         retention_dict[program] = retention_rate
+
   
     return retention_dict
 
```

Comment:
there is a huge piece of code with two functions, it doesn't look clean. Do you want to either use it or get rid of it?

Reply:
After verbal confirmation with Karina, she was referring to the commented code